docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :feature:`-` rotki will now properly decode the transactions for bridging to and from Scroll.
+* :feature:`7708` rotki now properly decode WETH transactions on all supported EVM chains with native ETH
```

Comment:
Seems this does not do it for non-native, but I would just add the changelog when it is **all done**. And not 1 changelog per entry.

```suggestion
* :feature:`7708` rotki now properly decodes all native to/from wrapped token transactions (e.g. ETH<->WETH, Matic<->WMatic) on all supported EVM chains
```